test/hotspot/jtreg/compiler/c2/x86/TestGetAndAdd.java

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Please, use `c2/varhandle` directory for this test. We don't use platform in dir names.
The test could be updated later for other platforms.